config/application.rb

```diff
@@ -1,50 +1,46 @@
-require File.expand_path('../boot', __FILE__)
+require_relative "boot"
 
 require "rails"
 # Pick the frameworks you want:
-require "active_storage/engine"
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
+require "active_storage/engine"
```

Comment:
Do we want to require these (i'm assuming, large) chunks of rails code if we're not using them? I imagine they don't have cross-dependencies.

Reply:
> Do we want to require these

Good question.

Probably not, but I also don't feel like investigating them all to see what they do 😅 .

I commented out `active_storage` and `action_text` because I know we're not using them.

I _uncommented_ `action_cable` because I want to start adding that in, and I know I'll forget to check here to see if it's required.